semantic-conventions/README.md

```diff
@@ -125,12 +125,66 @@ This way, multiple files are generated. The value of `pattern` can be one of the
 - `semconv_id`: The id of the semantic convention.
 - `prefix`: The prefix with which all attributes starts with.
 - `extends`: The id of the parent semantic convention.
+- `root_namespace`: The root namespace of attribute to group by.
```

Comment:
future: we might want to support more flexible patterns to generate files using multiple segments of namespace (e.g. `HttpClientMetrics`).